corehq/apps/app_manager/static/app_manager/js/details/case_claim.js

```diff
@@ -361,6 +361,21 @@ hqDefine("app_manager/js/details/case_claim", function () {
             );
         };
 
+        self.commonProperties = ko.computed(function () {
```

Comment:
[intersect](https://underscorejs.org/#intersection) might be useful in this function

Reply:
The objects inside have different structure, we have to intersect by `name` attr of one with the `'property` attr of other, so I went for the manual method here.